src/app/ubs-admin/components/ubs-admin-export-details/ubs-admin-export-details.component.scss

```diff
@@ -0,0 +1,64 @@
+* {
+  font-family: var(--ubs-quaternary-font);
+}
+
+@mixin input {
+  height: 36px;
+  left: calc(50% - 340px / 2);
+  top: calc(50% - 36px / 2 + 10px);
```

Comment:
It looks a bit strange
Why do we need such complicated calculations here?
You can use just
`left: calc(50% - 170px);`
`top: calc(50% - 28px);`

Reply:
Fixed